webserver/views/api/v1/datasets.py

```diff
@@ -217,7 +217,18 @@ def add_recordings(dataset_id):
 
     :resheader Content-Type: *application/json*
     """
-    raise NotImplementedError
+    class_dict = request.get_json()
+    if not class_dict:
+        raise api_exceptions.APIBadRequest("Data must be submitted in JSON format.")
```

Comment:
I have added a test for valid UUIDs, however, while creating a json request while testing, it automatically checks whether the recordings are a valid UUID. Does it need to be checked again in the view? The same goes for duplicate recordings in the JSON. While making the json request itself the program throws an error if there are duplicate recordings in the JSON.